graphql-test-utils/graphql-models/application-navbar-menu-group.ts

```diff
@@ -0,0 +1,12 @@
+import { faker } from '@faker-js/faker';
+import { Factory } from 'rosie';
+import ApplicationNavbarMenu from './application-navbar-menu';
+
+const ApplicationNavbarMenuGroup = new Factory()
```

Comment:
FYI: just so that we don't forget.

These (legacy) models should be either moved to the test-data repo or be defined using the test-data core package.

Reply:
Thank you for the feedback! I've refactored the added model to use `@commercetools-test-data/core` (and left the original models unchanged).

I've also updated the faker package version and applied faker methods throughout as needed.